src/Core/Localization/Currency/PatternTransformer.php

```diff
@@ -92,6 +92,29 @@ public function transform(string $currencyPattern, string $transformationType):
         return implode(';', $transformedPatterns);
     }
 
+    /**
+     * @param string $currencyPattern
+     *
+     * @return string
+     */
+    public function getTransformationType(string $currencyPattern)
+    {
+        $patterns = explode(';', $currencyPattern);
+        $pattern = str_replace(self::RTL_CHARACTER, '', $patterns[0]);
+
+        if (preg_match("/^¤[ \u{a0}].+/", $pattern)) {
```

Comment:
Afaik, you created a const for the `\u{a0}`

Reply:
For someone who don't know what `\u{a0}` is, it's pretty eassier to see `self::NO_BREAK_SPACE` :D